frontend/src/routes/edit/musics/+page.svelte

```diff
@@ -0,0 +1,151 @@
+<!--
+SPDX-FileCopyrightText: 2023 Marlon W (Mawoka)
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+<script lang="ts">
+	import BrownButton from '$lib/components/buttons/brown.svelte';
+	import { navbarVisible } from '$lib/stores';
+	import { getLocalization } from '$lib/i18n';
+	const { t } = getLocalization();
+
+	let file_input: HTMLInputElement;
+	navbarVisible.set(false);
+
+	let stats: { progress: number; time_elapsed: number; speed: number } = {
+		progress: 0,
+		time_elapsed: 0,
+		speed: 0
+	};
+
+	let upload_stats: { progress: number; upload_started: undefined | Date; time_elapsed: number } =
+		{
+			progress: 0,
+			upload_started: undefined,
+			time_elapsed: 0
+		};
+
+	const speed_extraction_regex = /speed=(\d+\.\d+)x(?![\s\S]*speed=\d+\.\d+x)/;
+	let last_time_elapsed = 0;
+
+	// eslint-disable-next-line no-unused-vars
+	enum Status {
+		// eslint-disable-next-line no-unused-vars
+		Idle,
+		// eslint-disable-next-line no-unused-vars
+		Compressing,
+		// eslint-disable-next-line no-unused-vars
+		CompressDone,
+		// eslint-disable-next-line no-unused-vars
+		Uploading,
+		// eslint-disable-next-line no-unused-vars
+		Done
+	}
+
+	let status: Status = Status.Idle;
+
+	let file_size_in_mi: undefined | number = undefined;
+	let original_file_size_in_mi: undefined | number = undefined;
+	let file: File;
+	let file_data: undefined | Blob = undefined;
+
+	const compress_music = async () => {
```

Comment:
What's this purpose?

Reply:
None, but I didn't manage and spend time to remove the way it used to work. So I kept the compression function if compression could be added.